src/client.cpp

```diff
@@ -688,4 +688,5 @@ void PauseStream(bool _UNUSED(bPaused)) {}
 bool SeekTime(int, bool, double*) { return false; }
 void SetSpeed(int) {};
 bool IsTimeshifting(void) { return false; }
+bool IsRealTimeStream(void) { return true; }
```

Comment:
IsTimeshifting alsways returning false in conjunction with IsRealTimeStream always returning true makes sence, but I doubt that IsTimeshifting is implemented correctly (yeah I know that this was not changed by this PR) as  implementation of the other timeshifting functions implies that this addon actually supports timeshifting => example: https://github.com/kodi-pvr/pvr.dvbviewer/pull/28/files#diff-34f48ad91ac6c202ac60b0348ae90e30R580

So, if this addon actually supports timeshifting, imo IsTimeshifting and IsRealTimeStream both need to be fixed to take actual timeshifting state into account.

@FernetMenta any opinions on that?